src/deploy-actions.js

```diff
@@ -179,6 +179,11 @@ async function deployWsk (scriptConfig, manifestContent, logFunc, filterEntities
     namespace: scriptConfig.ow.namespace
   }
 
+  // TODO: remove this once the feature flag is removed
```

Comment:
make sure to file a story for all the instances of the feature flag or code logic that needs to be removed